script.js

```diff
@@ -1,4 +1,312 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+var deck = [];
+var playerHand = [];
+var computerHand = [];
+var playerScore;
+var computerScore;
+var displayPlayerHand = [];
+var displayComputerHand = [];
+var winner = "";
+var input;
+
+// Draw a card
+var drawCard = function (numCards, hand) {
+  for (var i = 0; i < numCards; i += 1) {
+    var newCard = deck.pop();
+    hand.push(newCard);
+  }
+  return newCard;
+};
+
+// Display card
+var displayCard = function (hand) {
+  var showCard = [];
+  for (var i = 0; i < hand.length; i += 1) {
+    showCard.push(hand[i].name);
+  }
+  return showCard;
+};
+
+// Calculate score
+var calculateScore = function (hand) {
+  var score = 0;
+  for (var i = 0; i < hand.length; i += 1) {
+    score = score + hand[i].value;
+  }
+  return score;
+};
+
+// Create & shuffle the deck
+var startButton = document.getElementById("start-button");
+startButton.addEventListener("click", function () {
+  deck = [];
+  playerHand = [];
+  computerHand = [];
+  displayPlayerHand = [];
+  displayComputerHand = [];
+
+  var unshuffledDeck = makeDeck();
+  var shuffledDeck = shuffleCards(unshuffledDeck);
+  deck = shuffledDeck;
+  console.log("fresh deck", deck.length);
+
+  // Draws 2 cards for player
+  var playerDraws = drawCard(2, playerHand);
+  displayPlayerHand = displayCard(playerHand);
+  playerScore = calculateScore(playerHand);
+
+  // Draws 2 cards for computer
+  var computerDraws = drawCard(2, computerHand);
+  displayComputerHand = displayCard(computerHand);
+  computerScore = calculateScore(computerHand);
+
+  console.log("Player's first hand", playerHand);
+  console.log("Computer's first hand", computerHand);
+  console.log("First hand, player's score", playerScore);
+  console.log("First hand, computer's score", computerScore);
+
+  // Message
+  var output = document.getElementById("output-div");
+
+  if (playerScore < 21 && computerScore < 21) {
+    output.innerHTML = `1st round <br><br> Your hand: ${displayPlayerHand} <br> Score: ${playerScore} <br><br> Computer's hand: ${displayComputerHand[0]}, second card (face down). <br><br> Number of cards in deck: ${deck.length} <br><br> To continue, press Hit or Stand.`;
```

Comment:
It seems that HTML can display arrays for you quite well but I would not rely on html to display an array. Instead, your displayCard function should return a nicely formatted string. As to the hiding of one card of the computer, consider passing in an argument into the displayCard function to decide how many cards to display